cfme/tests/middleware/test_middleware.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from utils.browser import ensure_browser_open
+from cfme.login import login_admin
+from cfme.fixtures import pytest_selenium as sel
+
+
+@pytest.fixture
+def webDriver(request):
+    ensure_browser_open()
+    login_admin()
+
+    def closeSession():
+        """ Place Holder for after-test action """
+    request.addfinalizer(closeSession)
+
+    return
+
+
+def test_middleware():
+    sel.force_navigate('middleware')
```

Comment:
So, if you wanted to use the "fixture" you have defined above, you would need to put it in the constructor
ie
```python
def test_middleware(webDriver):
```
However, force_nagivate is special, it will open the browser and login, if the system doesn't already have an open logged in session. So in this particular case, the fixture is not needed.